chapters/synchronization.txt

```diff
@@ -894,8 +894,8 @@ endif::VK_NV_ray_tracing[]
 |ename:VK_ACCESS_TRANSFER_WRITE_BIT                           | ename:VK_PIPELINE_STAGE_TRANSFER_BIT
 |ename:VK_ACCESS_HOST_READ_BIT                                | ename:VK_PIPELINE_STAGE_HOST_BIT
 |ename:VK_ACCESS_HOST_WRITE_BIT                               | ename:VK_PIPELINE_STAGE_HOST_BIT
-|ename:VK_ACCESS_MEMORY_READ_BIT                              | N/A
-|ename:VK_ACCESS_MEMORY_WRITE_BIT                             | N/A
+|ename:VK_ACCESS_MEMORY_READ_BIT                              | Any
+|ename:VK_ACCESS_MEMORY_WRITE_BIT                             | Any
```

Comment:
Even `TOP_OF_PIPE` and `BOTTOM_OF_PIPE` ?

Reply:
Ugh my two least favourite pipeline stages... that's a good question. I'm going to say yes for now, but this probably needs affirmation by the WG.

Reply:
Oh and can `VK_ACCESS_MEMORY_WRITE_BIT` be on any stage that is read only, and vice versa?

Reply:
That one I don't think is contentious - that's already true from the current language IMO

Reply:
Just asking. It makes it different from the other access masks. So making sure that is intentional.
E.g. `VK_PIPELINE_STAGE_VERTEX_INPUT_BIT` otherwisely only accepts `VK_ACCESS_*_READ_BIT`. So `VK_ACCESS_MEMORY_WRITE_BIT` is valid there anyway?

Reply:
Vulkan WG agreed with my assertions - these are wildcards, valid anywhere you can stick an access flag.